packaging/helm/skipper-aws/templates/external-dns/deployment.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: external-dns
+  namespace: kube-system
+  labels:
+    application: external-dns
+    version: {{ .Values.externalDnsVersion }}
+spec:
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      application: external-dns
+  template:
+    metadata:
+      labels:
+        application: external-dns
+        version: {{ .Values.externalDnsVersion }}
+{{- if .Values.kube2iam }}
+      annotations:
+        iam.amazonaws.com/role: "{{ .Values.externalDnsAwsRole }}"
+{{- end }}
+    spec:
+      dnsConfig:
+        options:
+          - name: ndots
+            value: "1"
+      priorityClassName: system-cluster-critical
+      serviceAccountName: external-dns
+      containers:
+      - name: external-dns
+        image: registry.opensource.zalan.do/teapot/external-dns:{{ .Values.externalDnsVersion }}
```

Comment:
I recommend moving the image to a values property also as I know people (us included) pull all images via a local proxy (e.g. Artifactory) so would want to be able to override this.

Reply:
makes sense, will change it

Reply:
changed